recipe/meta.yaml

```
@@ -1,20 +1,20 @@
-{% set version = "2.10.0" %}
-{% set estimator_version = "2.10" %}
+{% set version = "2.11.0" %}
+{% set estimator_version = "2.11" %}
 
 package:
   name: tensorflow-split
   version: {{ version }}
 
 source:
   - url: https://github.com/tensorflow/tensorflow/archive/refs/tags/v{{ version.replace(".rc", "-rc") }}.tar.gz
-    sha256: b5a1bb04c84b6fe1538377e5a1f649bb5d5f0b2e3625a3c526ff3a8af88633e8
+    sha256: 99c732b92b1b37fc243a559e02f9aef5671771e272758aa4aec7f34dc92dac48
     patches:
       - patches/0001-loosen-requirements.patch
-      - patches/0002-cublas-fix-backport.patch
 
-
-  - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ estimator_version.replace(".rc", "-rc") }}.tar.gz
-    sha256: 60df309377cf4e584ca20198f9639beb685d50616395f50770fc0999092d6d85
+  # they didn't release estimator 2.11, but it's needed.. 
+  # - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ estimator_version.replace(".rc", "-rc") }}.tar.gz
+  - url: http://github.com/tensorflow/estimator/archive/c7e88f0b110aa1b352e2af3e8876e797a94c27a3.tar.gz
+    sha256: 5275359ff105742016d7a4c0a01b23eb21b30106b11b0b7f27f4cc75bc634726
```

Released now: https://github.com/tensorflow/estimator/releases/tag/v2.11.0
```suggestion
  - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ estimator_version.replace(".rc", "-rc") }}.tar.gz
    sha256: 922f9187de79e8e7f7d7a5b2d6d3aabc81bbbd6ba5f12a4f52967dd302214a43
```